File: autoit-mediapipe-com/src/binding/tasks/components/containers/bounding_box.cc
#include "binding/tasks/components/containers/bounding_box.h"

namespace mediapipe::tasks::components::containers {
	LocationData::BoundingBox ConvertRectToProto(Rect* rect) {
		LocationData::BoundingBox bounding_box_proto;
		bounding_box_proto.set_xmin(rect->left);
		bounding_box_proto.set_ymin(rect->top);
		bounding_box_proto.set_width(rect->right - rect->left);
		bounding_box_proto.set_height(rect->bottom - rect->top);
		return bounding_box_proto;
	}
}

File: src/declarations/tasks/components/containers/bounding_box.js
module.exports = ({language, self_get}) => [
    // expose bounding_box properties like in mediapipe python
    [`mediapipe.tasks.${ language }.components.containers.bounding_box.`, "", ["/Properties"], [
        ["mediapipe::tasks::components::containers::Rect", "BoundingBox", "", ["/R", "=this", "/S"]],
    ], "", ""],
];
